openbb_platform/providers/oecd/openbb_oecd/models/gdpforecast.py

```diff
@@ -0,0 +1,73 @@
+from datetime import date
+from typing import Any, Dict, List, Literal, Optional
+
+from openbb_oecd.utils import constants, helpers
+from openbb_provider.abstract.fetcher import Fetcher
+from openbb_provider.standard_models.gdpforecast import (
+    GDPForecastData,
+    GDPForecastQueryParams,
+)
+from pydantic import Field
+
+gdp_countries = tuple(constants.COUNTRY_TO_CODE_GDP_FORECAST.keys())
+
+
+class OECDGDPForecastQueryParams(GDPForecastQueryParams):
+    """GDP Forecast query."""
+
+    country: Literal[*gdp_countries] = Field(
+        description="Country to get GDP for.", default="united_states"
+    )
+
+
+class OECDGDPForecastData(GDPForecastData):
+    """GDP Forecast data from OECD."""
+
+
+class OECDGDPForecastFetcher(
+    Fetcher[OECDGDPForecastQueryParams, List[OECDGDPForecastData]]
+):
+    """OECD GDP Forecast Fetcher."""
+
+    @staticmethod
+    def transform_query(params: Dict[str, Any]) -> OECDGDPForecastQueryParams:
+        if params["start_date"] is None:
```

Comment:
We are using `transformed_params` internally for consistency so might modify this to adhere to it.

Reply:
fineeeeeee